venvUtils/ensureAndActivate.bat

```diff
@@ -1,9 +1,19 @@
 @echo off
 rem this script ensures the NVDA build system Python virtual environment is created and up to date,
 rem and then activates it.
-rem this script should be used only in the case where many commands will be executed within the environment and the shell will be eventually thrown away. 
-rem E.g. an Appveyor build.
+rem This is an internal script and should not be used directly.
+
+rem Ensure the environment is created and up to date
 py -3.8-32 "%~dp0\ensureVenv.py"
 if ERRORLEVEL 1 goto :EOF
-call "%~dp0\..\.venv\scripts\activate.bat"

```

Comment:
It would be good to have comments that delineate the following set of steps as having been adapted from the `activate.bat` script. This will help us to understand their origin and how to update them if they stop working (eg the requirements of the `activate.bat` script change).